src/components/credit-card-date/scripts/build-dist.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ -z "$VARIANT" ]; then VARIANT=''; fi
+if [ -z "$TEMPLATE_LANG" ]; then TEMPLATE_LANG='en'; fi
+
+echo "Building package for demo. Variant: '$VARIANT', language: '$TEMPLATE_LANG'."
+
+browserify -t [ redirectify --dir "$VARIANT" ] -t reactify index.js --standalone ui-component-creditcard > dist/ui-component-creditcard-standalone.js
```

Comment:
if we're keeping this then we should probably change `ui-component-creditcard` to `ui-component-credit-card-date`